locales/yarr.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Execute binaries from npm packages.\n%s": "Set sail yer bins fro' npm treasures.\n%s",
+  "Package to be installed.": "Treasures t' be looted.",
+  "Location of the npm cache.": "Locale o' th' buried npm cache.",
+  "Skip installation if a package is missing.": "Skip lootin' if th' treasure's been theived.",
+  "Path to user npmrc.": "Map t' scallywag npmrc.",
+  "Execute string as if inside `npm run-script`.": "Follow the map like ye be in `npm run-script`.",
```

Comment:
Would be pretty cool to retain "as if" since I think "like" is a pretty modern thing.
"Follow the map as if ye be in `npm run script`."